setup.py

```diff
@@ -70,7 +70,7 @@
     },
     install_requires=[
         "numpy >= 1.14.5",
-        "protobuf >= 3.1.0, <= 3.20.1",
+        "protobuf >= 3.1.0, <= 4.0.0",
```

Comment:
We want `, < 4.0.0` not `, <= 4.0.0`.

`4.0.0` contains a breaking change.

Reply:
fixed!